features/org.wso2.carbon.siddhi.extensions.installer.core.feature/src/main/resources/extensionsInstaller/extensionDependencies.json

```diff
@@ -279,10 +279,10 @@
     ]
   },
 
-  "cdc": {
+  "cdcoracle": {
```

Comment:
Please change as `cdc-mysql`. Should be similar to the `name` property of `extension`

Reply:
Done.